src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/StartTask.groovy

```diff
@@ -26,6 +27,12 @@ class StartTask extends AbstractServerTask {
 
     protected final String START_APP_MESSAGE_REGEXP = "CWWKZ0001I.*"
 
+    StartTask() {
+        description 'Starts the Liberty server.'
```

Comment:
Why are lines 31-33 not inside configure() like so many of the other tasks? Same question for StatusTask.

Reply:
Ah, looks like I missed a couple. They should be in configure().